python/tests/expressions/test_visitors.py

```diff
@@ -827,85 +840,91 @@ def manifest_no_stats() -> ManifestFile:
     return _to_manifest_file()
 
 
+def _PartitionFieldSummary(
+    contains_null: bool, contains_nan: Optional[bool], lower_bound: Optional[bytes], upper_bound: Optional[bytes]
+) -> PartitionFieldSummary:
+    return PartitionFieldSummary.from_record(Record(contains_null, contains_nan, lower_bound, upper_bound))
+
+
 @pytest.fixture
 def manifest() -> ManifestFile:
     return _to_manifest_file(
         # id
-        PartitionFieldSummary(
+        _PartitionFieldSummary(
```

Comment:
Is it worth adding the `_` prefix? For records from the spec, I'd probably omit it.